end_to_end/signal_handler_test.go

```
@@ -11,6 +13,14 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+func CreateOutput(cmd *exec.Cmd) (*bufio.Scanner, strings.Builder) {
+	outPile, _ := cmd.StdoutPipe()
```

Did you mean outPipe?
IMHO we shall not ignore errors

---

We do not check for errors in a large number of places in the tests.

---

maybe create new separate issue to check all errors in all tests? (by `Expect(err).ToNot(HaveOccurred())` )

---

Yes, and it's a big tests shortage. We don't obliged to have it in a new code.

---

> maybe create new separate issue to check all errors in all tests? (by `Expect(err).ToNot(HaveOccurred())` )

good idea, but here we making new code, I suggest to make it better from it's beginning